lib/message.rb

```diff
@@ -0,0 +1,17 @@
+require 'twilio-ruby'
+require 'dotenv/load'
+
+class Message
+
+def initialize
+  @client = Twilio::REST::Client.new(ENV['SID'], ENV['TOKEN'])
```

Comment:
Right now, any time you create an instance of your Message class, you're hardwired to create an instance of the real Twilio client too. This is a creating a hard dependency between those objects and making it so that you'll send yourself a text message every time that you run your unit tests.

A simple fix for this would be to make use of dependency injection

```ruby
def initialize(client = Twilio::REST::Client.new(ENV['SID'], ENV['TOKEN']))
  @client = client
end
```